NAMESPACE

```diff
@@ -1,2 +1,23 @@
 # Generated by roxygen2: do not edit by hand
 
+export(PHGServerCon)
+export(brapiURL)
+export(brapiVersion)
+export(host)
+export(httProtocol)
+export(phgType)
+export(port)
+export(readRefRanges)
+export(readSamples)
+export(serverInfo)
+exportClasses(PHGCon)
+exportClasses(PHGServerCon)
+exportMethods(brapiURL)
+exportMethods(brapiVersion)
+exportMethods(host)
+exportMethods(httProtocol)
+exportMethods(phgType)
+exportMethods(port)
+exportMethods(readSamples)
+exportMethods(serverInfo)
```

Comment:
This is repeated

Reply:
Oh one says exportMethods

Reply:
Yes - `export()` is for exporting function calls into the R namespace and `exportMethods()` is for exporting any package defined generics that are not in the base libraries.